src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -49,8 +49,19 @@ public override void InitializeResources()
                return;
             _presenter.SaveConfiguration(fileDialog.FileName);
          });
-         saveMappingBtn.Text = Captions.Importer.SaveConfiguration;
-         saveMappingBtnLayoutControlItem.AdjustButtonSize();
+         applyMappingBtn.Click += (s, a) => OnEvent(() =>
+         {
+            var fileDialog = new OpenFileDialog{ };
```

Comment:
This should be in the presneter this stuff..,.using IDIalogCreator etc.. No logic like this in the view  as it helps with unit testing the behavior. We are weak on unit test at the moment on this feature but at least, we won't shoot ourselves in the foot

Reply:
moved to presenter